rioting-daisies/BangazonSite
# Product types

# Description

This PR creates a new view when navigating to product types, which shows the list of product types, and up to 3 products from the category. 



Fixes Issue #1 

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1. git fetch and checkout product-types
1. navigate to product types, products and Product categories should appear, and all links should be active and accurate. 


# Checklist:
